### PR TITLE
Add message for when user is not logged in.

### PR DESCRIPTION
**Pull request by imthinhvu**
- [x] I have put a reference to a related issue in the description
- [x] I have created the pull request from the correct version branch
- [x] I have ensured the branch is rebased and/or there are no merge conflicts

Addresses https://github.com/jyntran/nhaccuatui-scrobbler/issues/13.

Screen Shot 2019-10-17 at 1 41 10 AM

### CODE REVIEW

File: popup.html [hunk 1]
```diff
@@ -86,5 +89,4 @@ <h3>NhacCuaTui Scrobbler</h3>
     </div>
   </div>
   </body>
-</html>
```
Comment (jyntran) [inline popup.html]: Looks like the `loginPrompt` element got lost in the commits, please add it back in?

File: popup.html [hunk 2]
```diff
@@ -28,6 +28,9 @@
         background-image: linear-gradient(0deg, #fff, #fff, #f1f1f1);
         padding: 8px;
       }
+      #loginPrompt {
+        color: #c2a90f;
```
Comment (jyntran) [inline popup.html]: Looking at it now, dark yellow doesn't look as good as red. Sorry about that! I would make it `#cc0000` instead.

**Comment by jyntran** [comment]
@imthinhvu Also can this merge request point to my `v1.5` branch, if it's possible?

**Comment by imthinhvu** [comment]
@jyntran - I incorporated your comments and rebased to v1.5. Let me know if you require anything else.

**Pull request merged**